src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp

```diff
@@ -2108,7 +2108,7 @@ static void test24_summaryWithQueueDetailsTest()
     fields.push_back("Number of partially confirmed messages");
     fields.push_back("Number of confirmed messages");
     fields.push_back("Number of outstanding messages");
-    bmqu::AlignedPrinter printer(expectedStream, &fields);
+    bmqu::AlignedPrinter printer(expectedStream, fields);
```

Comment:
Also need to change type a few lines above:
`bsl::vector<const char*> fields` -> `bsl::vector<bsl::string> fields`